xml/System.Runtime.InteropServices/NativeMemory.xml

```diff
@@ -0,0 +1,302 @@
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig void AlignedFree(void* ptr) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Runtime.InteropServices.NativeMemory.AlignedFree(System.Void*)" />
+      <MemberSignature Language="F#" Value="static member AlignedFree : nativeptr&lt;unit&gt; -&gt; unit" Usage="System.Runtime.InteropServices.NativeMemory.AlignedFree ptr" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; static void AlignedFree(void* ptr);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime.InteropServices</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Attributes>
+        <Attribute>
+          <AttributeName Language="C#">[System.CLSCompliant(false)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.CLSCompliant(false)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="ptr" Type="System.Void*" />
+      </Parameters>
+      <Docs>
+        <param name="ptr">To be added.</param>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="AlignedRealloc">
+      <MemberSignature Language="C#" Value="public static void* AlignedRealloc (void* ptr, UIntPtr byteCount, UIntPtr alignment);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig void* AlignedRealloc(void* ptr, native unsigned int byteCount, native unsigned int alignment) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Runtime.InteropServices.NativeMemory.AlignedRealloc(System.Void*,System.UIntPtr,System.UIntPtr)" />
+      <MemberSignature Language="F#" Value="static member AlignedRealloc : nativeptr&lt;unit&gt; * unativeint * unativeint -&gt; nativeptr&lt;unit&gt;" Usage="System.Runtime.InteropServices.NativeMemory.AlignedRealloc (ptr, byteCount, alignment)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; static void* AlignedRealloc(void* ptr, UIntPtr byteCount, UIntPtr alignment);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime.InteropServices</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Attributes>
+        <Attribute>
+          <AttributeName Language="C#">[System.CLSCompliant(false)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.CLSCompliant(false)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <ReturnValue>
+        <ReturnType>System.Void*</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="ptr" Type="System.Void*" />
+        <Parameter Name="byteCount" Type="System.UIntPtr" />
```

Comment:
I think these should be `nuint`. Technically this is the same thing but from an API the `nuint` is the definition. @tannergooding

Reply:
This is likely a place where the docs tooling doesn't understand certain newer language features. Off the top of my head:
* nint/nuint types
* in parameters
* readonly members
* function pointers
* ref and ref readonly returns
* records
* tuple names
* init only setters

There are possibly more as well and its possible that a few of these have already been fixed.

Reply:
@TianqiZhang Can you comment on this?